crates/m3/Cargo.toml

```diff
@@ -9,8 +9,8 @@ anyhow.workspace = true
 binius_core = { path = "../core" }
 binius_field = { path = "../field" }
 binius_math = { path = "../math" }
-binius_maybe_rayon = { path = "../maybe_rayon" }
-binius_utils = { path = "../utils" }
+binius_maybe_rayon = { path = "../maybe_rayon", default-features = false }
```

Comment:
Shouldn't this crate now have a `rayon` or `parallel` feature? How do we build it with rayon?

Reply:
With this MR, it looks like the `m3` crate always gets built without rayon and there's no way to enable it. Is that not true?

Reply:
Ah, nevermind https://doc.rust-lang.org/cargo/reference/features.html#feature-unification.